trino-aws-proxy/src/main/java/io/trino/aws/proxy/server/rest/ResourceSecurity.java

```diff
@@ -75,5 +75,15 @@ public SigningServiceType signingServiceType()
         }
     }
 
+    final class Glue
```

Comment:
With the changes made in the previous PR, I would say this should live in the Glue module